src/content/reference/react-dom/components/form.md

```diff
@@ -0,0 +1,445 @@
+---
+title: "<form>"
+canary: true
+---
+
+<Canary>
+
+React's `form` Component is currently only available in React's canary and experimental channels but you can use the [standard HTML `form` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) in your React's stable channel. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+
+<Intro>
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `<form>` {/*option*/}
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+[See more examples below.](#usage)
+
+#### 🚧 Props {/*props*/}
+
+`<form>` supports all [common element props.](/reference/react-dom/components/common#props)
+
+Additionally, `<form>` supports these props:
+
+* [`action`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#disabled): A boolean. Specifies what to do when a form is submitted. The form data can be handled with client or server side code.
```

Comment:
Should this link to the [form action attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes_for_form_submission) instead of the html option?

Reply:
The type is wrong, it should say 'a function' instead of 'a boolean'.

Reply:
Thanks, fixed both issues